d2game/d2player/escape_menu.go

```diff
@@ -427,22 +418,10 @@ func (m *EscapeMenu) showLayout(id layoutID) {
 	}
 
 	if id == saveLayoutID {
-		// Save Game
-		heroState := d2hero.HeroState{
-			HeroName:   m.gameClient.Players[m.gameClient.PlayerID].Name(),
-			HeroType:   m.gameClient.Players[m.gameClient.PlayerID].Class,
-			HeroLevel:  m.gameClient.Players[m.gameClient.PlayerID].Stats.Level,
-			Act:        1,
-			FilePath:   "",
-			Equipment:  *m.gameClient.Players[m.gameClient.PlayerID].Equipment,
-			Stats:      m.gameClient.Players[m.gameClient.PlayerID].Stats,
-			Skills:     m.gameClient.Players[m.gameClient.PlayerID].Skills,
-			X:          m.gameClient.Players[m.gameClient.PlayerID].Position.X(),
-			Y:          m.gameClient.Players[m.gameClient.PlayerID].Position.Y(),
-			LeftSkill:  m.gameClient.Players[m.gameClient.PlayerID].LeftSkill,
-			RightSkill: m.gameClient.Players[m.gameClient.PlayerID].RightSkill,
+		err := m.player.Save()
+		if err != nil {
+			return
```

Comment:
Minor nitpick. Can you print the error here, instead of silently failing?

Reply:
I love nitpicks. Yes, printing it is!